docs/externalGrading.md

```diff
@@ -258,44 +258,51 @@ In production, PrairieLearn runs external grading jobs on a distributed system t
 
 PrairieLearn supports two ways of running on your own machine: natively, and inside a Docker container. We support running external autograders for each way. If the `HOST_JOBS_DIR` environment variable is set (more on that later), PrairieLearn will assume it's running in a container; otherwise, it assumes it's running natively.
 
-#### Running locally (on Docker)
+### Running locally (on Docker)
```

Comment:
Not requesting a change yet, but: I've thought for a while that burying this information on the external grading page is not the best; perhaps this should be present on `installing.md` instead?

Reply:
That is actually a good idea. The workspace page refers to it as well anyway, might as well have both pointing to the same doc. Will change.

Reply:
Done in 720b5e1f6.